examples/direct_fidelity_estimation.py

```diff
@@ -52,54 +58,108 @@ def compute_characteristic_function(circuit: cirq.Circuit,
     return trace, prob
 
 
+def sample_pauli_states(n_qubits: int, n_trials: int, physical: bool):
+    pauli_gates = [cirq.I, cirq.X, cirq.Y, cirq.Z]
+    if physical:
```

Comment:
`physical` is not a very descriptive name. For example, it seems to control whether or not you are sampling pauli strings to try instead of trying all of them. Maybe `sampled`? `sparse`?

Reply:
Changed to sampled.